src/components/common/ConfirmModal.tsx

```diff
@@ -38,6 +38,11 @@ const dialogStyles = makeStyles({
     height: 48,
     width: 107,
   },
+  dialogAction: {
```

Comment:
this is identical to `dialogCancel`, perhaps they can just be merged into 1 obj
```suggestion
  dialogButton: {
```